spec/support/paper_trail_helper.rb

```diff
@@ -0,0 +1,13 @@
+# enable PaperTrail for specific tests - you can add a with_versioning test helper method
```

Comment:
**question:** again, is this necessary? I think `with_versioning` is built in to `paper_trail` and you can also add `versioning: true` to the test definition, i.e. https://github.com/18F/identity-dashboard/blob/main/spec/models/user_team_spec.rb#L4.